### ISSUE TITLE
A11y: Expandables (section, panel, trigger) lack aria properties

### ISSUE THREAD

**Issue by ffriedl89** [dynatrace-oss/barista#788]
# Bug Report

The expandable trigger lacks the correct aria attributes.

## Expected Behavior

Expandable trigger: 
The trigger should have the `aria-controls` property set to the id of the expandable panel it controls.
The trigger should also have the `aria-expanded` set depending on the expand state of the panel. 

This is also expected to behave correctly in the expandable panel, but this should be covered since we use the trigger internally.

for references and a11y best practices please see https://www.w3.org/TR/wai-aria-practices-1.1/#accordion
https://www.w3.org/TR/wai-aria-1.1/#aria-expanded

## Current Behavior

Currently there is no aria-expanded or aria-controls property set on the trigger.

## Possible Solution

Add those properties to the host metadata. We might also have to introduce an id input for the expandable panel with a generated id defaulted similar to the id we have in the radio.ts for example to be
